### ISSUE TITLE
Map widget focus shifting

### ISSUE THREAD

**Issue by johnatawnclementawn** [archesproject/arches#9851]
Address missing focus shifting in the map widget as pointed out in the [review of #9488](https://github.com/archesproject/arches/pull/9488#pullrequestreview-1495756724)
